WordPress/Classes/Services/SignupService.swift

```diff
@@ -136,11 +136,42 @@ open class SignupService: LocalCoreDataService {
                                             andClientSecret: ApiCredentials.secret(),
                                             success: { (responseDictionary) in
                                                 // Note: User creation is deferred until we have a WPCom auth token.
-                                                success()
+//                                                signin
                                             },
                                             failure: failure)
     }
 
+    /*
+
+ */
+    func createWPComeUserWithGoogle(token: String,
```

Comment:
1) This comment block is empty. 2) I can't believe you used the old-school style! 😉

Reply:
Ah this and the previous one where changes I didn't push :( 

Thanks for spotting them!